plugins/ui/src/js/src/elements/usePickerProps.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  SerializedFocusEventCallback,
+  useFocusEventCallback,
+} from './spectrum/useFocusEventCallback';
+import {
+  SerializedKeyboardEventCallback,
+  useKeyboardEventCallback,
+} from './spectrum/useKeyboardEventCallback';
+
+export interface SerializePickerEventProps {
```

Comment:
Will need to update the references to this as well.
```suggestion
export interface SerializedPickerEventProps {
```